feature/client/src/main/java/com/mifos/feature/client/createNewClient/CreateNewClientScreen.kt

```diff
@@ -403,109 +414,44 @@ private fun CreateNewClientContent(
-                .fillMaxWidth()
-                .padding(vertical = 16.dp),
-        ) {
-            Image(
-                painter = if (selectedImageUri.path?.isNotEmpty() == true) rememberAsyncImagePainter(
-                    selectedImageUri
-                ) else painterResource(
-                    id = R.drawable.feature_client_ic_dp_placeholder
-                ),
-                contentDescription = null,
-                modifier = Modifier
-                    .align(Alignment.Center)
-                    .clickable {
-                        showImagePickerDialog = true
-                    }
-                    .border(color = DarkGray, width = 2.dp, shape = CircleShape)
-                    .size(80.dp)
-                    .clip(CircleShape)
-            )
+        ClientHeader()
+        ClientImageSection(selectedImageUri = selectedImageUri) {
+            showImagePickerDialog = true
         }
-
-        Spacer(modifier = Modifier.height(16.dp))
-
-        MifosOutlinedTextField(
-            value = firstName,
-            onValueChange = { firstName = it },
-            label = stringResource(id = R.string.feature_client_first_name_mandatory),
-            error = null
-        )
-
-        Spacer(modifier = Modifier.height(16.dp))
-
-        MifosOutlinedTextField(
-            value = middleName,
-            onValueChange = { middleName = it },
-            label = stringResource(id = R.string.feature_client_middle_name),
-            error = null
+        ClientInputTextFields(
+            firstName = firstName,
+            middleName = middleName,
+            lastName = lastName,
+            mobileNumber = mobileNumber,
+            externalId = externalId,
+            onFirstNameChange = { firstName = it },
+            onMiddleNameChange = { middleName = it },
+            onLastNameChange = { lastName = it },
+            onMobileNumberChange = { mobileNumber = it },
+            onExternalIdChange = { externalId = it },
         )
-
-        Spacer(modifier = Modifier.height(16.dp))
-
-        MifosOutlinedTextField(
-            value = lastName,
-            onValueChange = { lastName = it },
-            label = stringResource(id = R.string.feature_client_last_name_mandatory),
-            error = null
-        )
-
-        Spacer(modifier = Modifier.height(16.dp))
-
-        MifosOutlinedTextField(
-            value = mobileNumber,
-            onValueChange = { mobileNumber = it },
-            label = stringResource(id = R.string.feature_client_mobile_no),
-            error = null,
-            keyboardType = KeyboardType.Number
-        )
-
-        Spacer(modifier = Modifier.height(16.dp))
-
-        MifosOutlinedTextField(
-            value = externalId,
-            onValueChange = { externalId = it },
-            label = stringResource(id = R.string.feature_client_external_id),
-            error = null
-        )
-
-        Spacer(modifier = Modifier.height(16.dp))
-
```

Comment:
Is there any reason why we removing this from here?

Reply:
ok. did you test the funcionality? Everything works?

Reply:
Yes, functionality has been preserved